modules/plugin-search.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 use Automattic\Jetpack\Constants;
+use Automattic\Jetpack\Partner;
```

Comment:
Good news! It looks like we can remove the whole `get_upgrade_url` function since 5308f676f40d22b44c5999e3f5fc72bfe8eb1524, it isn't used anywhere anymore. It's my bad, I should have removed it back then, when removing the only use of it. 

You can safely delete it, thus making this file a bit simpler.